frontend/packages/console-shared/src/components/close-button/CloseButton.scss

```diff
@@ -1,12 +1,5 @@
-.co-close-button {
-
-  &--float-right {
```

Comment:
`--float-right` is no longer needed as PatternFly's `<CloseButton>` has `float: inline-end;` set on it.